app/templates/application/dashboard.html

```diff
@@ -33,9 +37,9 @@ <h2 class="ui header">
                 <tbody>
                 {% for a in applicants %}
                     <tr>
-                        <td>{{ a.first_name }}</td>
-                        <td>{{ a.last_name }}</td>
-                        <td>{{ a.application.phone_number }}</td>
+                        <td>{{ user[loop.index0].first_name }}</td>
```

Comment:
This breaks the route `/applications` – can we change this to `a.first_name`, and change the join query `views.py` to a query for Users whose application is not None?

Reply:
This is the specific issue: #34